### DISCUSSION TITLE
Using A1111 WebUI I'm not able to see the TAP Dreambooth

### DISCUSSION THREAD

**Discussion by Faldo79** [d8ahazard/sd_dreambooth_extension#1080] [Q&A]
I've just installed the last version of A1111 (I'm starting with the AI). And it works to generate images using 1.5 and 2.1 models. 

But I've installed the Dreambooth, restart the UI and the Dreambooth tap doesn't show. The WebUI returned some errors. 

> 
ModuleNotFoundError: No module named 'diffusers'
Error loading script: main.py

  File "C:\A1111\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\utils\image_utils.py", line 11, in <module>
    from diffusers.schedulers import KarrasDiffusionSchedulers
ModuleNotFoundError: No module named 'diffusers'

Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Error executing callback app_started_callback for C:\A1111\stable-diffusion-webui\extensions\sd_dreambooth_extension\scripts\api.py
Traceback (most recent call last):
  File "C:\A1111\stable-diffusion-webui\modules\script_callbacks.py", line 107, in app_started_callback
    c.callback(demo, app)
  File "C:\A1111\stable-diffusion-webui\extensions\sd_dreambooth_extension\scripts\api.py", line 524, in dreambooth_api
    model_cfg: DreamboothConfig = Body(description="The config to save"),
NameError: name 'DreamboothConfig' is not defined

> 
Any ideas? 

thank you

**Comment by Faldo79** [comment]
Ok 

I have install diffuses and add the command "REQS_FILE=.\extensions\sd_dreambooth_extension\requirements.txt" to the user.bat. 

The Dreamboot tap is already there.

**Comment by InushaDeSilva** [comment]
Try updating the extension, the issue was fixed with the latest merge
